docs/LanguageGuide.md

```diff
@@ -69,6 +69,36 @@ foo ; symbol
 (defmodule <name> <definition1> <definition2> ...) ;; The main way to organize your program into smaller parts
 ```
 
+### Conditional statements with 'cond'
+The cond statement executes a block of code if a specified condition is true. If the condition is false, another block of code can be executed. Examples are given below:
+
+```clojure
```

Comment:
I’m not sure we need to give the whole definition of `cond` here. I think more examples might be more useful!

OTOH it might be instructive for people to understand how `cond works, so I’m not sure.

Reply:
I put in a few more examples. Shall be glad to make changes once you check into it.

Reply:
Yeah, the definition of cond-internal should not be here.